<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_PlayerPreferences</class>
 <widget class="QWidget" name="GUI_PlayerPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>115</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QCheckBox" name="cb_update_notifications">
     <property name="text">
      <string>Update notifications</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="cb_start_in_tray">
     <property name="text">
      <string>Start in tray</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="cb_close_to_tray">
     <property name="text">
      <string>Close to tray</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="cb_show_tray_icon">
     <property name="text">
      <string>Show system tray icon</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
